src/components/S/IStamp.vue

```diff
@@ -0,0 +1,80 @@
+<script setup lang="ts">
+import { getUrl } from '@/helpers/utils';
+
+const props = defineProps<{
+  modelValue?: string;
+  error?: string;
+  definition: any;
+}>();
+
+const emit = defineEmits<{
+  (e: 'update:modelValue', value: string);
+}>();
+
+const { upload, isUploadingImage } = useImageUpload();
```

Comment:
Seems a bit weird, that isUploadingImage is part of the composable, but uploadSuccess isn't.